src/dateparser/test/dateparser.spec.js

```diff
@@ -790,5 +790,27 @@ describe('date parser', function() {
         expect(toWestDate.getTime() - toEastDate.getTime()).toEqual(1000 * 60 * 60 * 12);
       });
     });
+
+    describe('overrideParser', function() {
+      it('overrides the parser', function() {
+        var original = dateParser.getParser('yy');
+        try {
+          expect(dateParser.parse('67', 'yy').getFullYear()).toEqual(2067);
+          expect(dateParser.parse('68', 'yy').getFullYear()).toEqual(2068);
```

Comment:
These two lines probably aren't necessary, as they should already be covered by tests for the format